python/tests/test_highlevel.py

```diff
@@ -2213,7 +2248,7 @@ def test_newick_topology_equiv(self):
         replace_numeric = {ord(x): None for x in "1234567890:."}
         for ts in get_example_tree_sequences():
             for tree in ts.trees():
-                if tree.num_roots > 1:
+                if not tree.has_single_root:
                     continue
```

Comment:
Shouldn't this be expecting an error rather than continuing?

Reply:
I think we do that in the simple `test_newick` function. Since we are testing topological equivalence here I thought that was irrelevant. But perhaps we need to be testing that *both* fail in an equivalent way.

Reply:
And, to follow-up, in pytest, how do I test between equivalence of 2 errors?

Reply:
Sorted this now. Hitting another issue :(